server/enums/applicationAction.ts

```diff
@@ -0,0 +1,13 @@
+enum ApplicationAction {
```

Comment:
An enum describing actions that a user can do in the application.
I _think_ this is most of them, but should be enough to get us going / serve as an example; and if we discover other actions that we need to protect with RBAC then this is where we will define them